R/fda-table_02.R

```diff
@@ -247,3 +247,72 @@ make_table_02_tplyr <- function(df,
   }
   tbl
 }
+
+
+#' @describeIn make_table_02 Create FDA table 2 using functions from `gtsummary`.
+#'
+#' @return
+#' * `make_table_02_gt` returns a `tbl_summary` object
+#'
+#' @examples
+#'
+#' advs <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "advs")
+#' adsl <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adsl") %>%
+#'   mutate(AGEGR1 = as.factor(case_when(
+#'     AGE >= 17 & AGE < 65 ~ ">=17 to <65",
+#'     AGE >= 65 ~ ">=65",
+#'     AGE >= 65 & AGE < 75 ~ ">=65 to <75",
+#'     AGE >= 75 ~ ">=75"
+#'   )) %>%
+#'     formatters::with_label("Age Group, years"))
+#'
+#' advs <- advs %>%
+#'   dplyr::filter(AVISIT == "BASELINE", VSTESTCD == "TEMP") %>%
+#'   dplyr::select("USUBJID", "AVAL")
+#'
+#' anl <- dplyr::left_join(adsl, advs, by = "USUBJID") %>%
+#'   formatters::var_relabel(
+#'     AGE = "Age, years",
+#'     AVAL = "Baseline Temperature (C)"
+#'   )
+#'
+#' tbl <- make_table_02_gt(df = anl)
+#' tbl
```

Comment:
It will use `adsl` as defined in the `make_table_02` example above.
```suggestion
#' tbl <- make_table_02_gt(df = adsl)
#' tbl
```